elfpy/eth/transactions.py

```diff
@@ -66,6 +66,56 @@ def smart_contract_read(contract: Contract, function_name: str, *fn_args, **fn_k
     return {f"value{idx}": value for idx, value in enumerate(return_values)}
 
 
+def smart_contract_preview_transaction(
+    contract: Contract, signer: EthAccount, function_name: str, *fn_args
+) -> dict[str, Any]:
+    """Return from a smart contract read call
+
+    Arguments
+    ---------
+    contract : web3.contract.contract.Contract
+        The contract that we are reading from.
+    function_name : str
+        The name of the function
+    *fn_args : Unknown
+        The arguments passed to the contract method.
+    **fn_kwargs : Unknown
+        The keyword arguments passed to the contract method.
+
+    Returns
+    -------
+    dict[str, Any]
+        A dictionary of value names
+    .. todo::
```

Comment:
I thnk you want a newline above the TODO so that sphinx will format it correctly